gap/PolygonalComplexes/animating.gd

```diff
@@ -0,0 +1,537 @@
+#! @Returns the updated print record.
+#! @Arguments surface, printRecord
+DeclareOperation( "ActivateEdges", [IsSimplicialSurface and IsNotEdgeRamified, IsRecord] );
+#! @Arguments surface, printRecord
+DeclareOperation( "DeactivateEdges", [IsSimplicialSurface and IsNotEdgeRamified, IsRecord] );
+#! @EndGroup
+
+#! @BeginGroup ActivateEdge
+#! @Description
+#! Activate the edge in position <A>i</A> of <A>Edges(surface)</A>. If an edge is active, then he is shown as a line in the animation. (For more information look at the start of the section <Ref Subsect="Section_LabelVisibility"/>)
+#! @Returns the updated print record.
+#! @Arguments surface, i, printRecord
+DeclareOperation( "ActivateEdge", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @Arguments surface, i, printRecord
+DeclareOperation( "DeactivateEdge", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @Arguments surface, i, printRecord
+DeclareOperation( "IsEdgeActive", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @EndGroup
+
+#! @BeginGroup ActivateFaces
+#! @Description
+#! Activate all faces. If a face is active, then the face is shown in the animation as an area. (For more information look at the start of the section <Ref Subsect="Section_LabelVisibility"/>)
+#! @Returns the updated print record.
+#! @Arguments surface, printRecord
+DeclareOperation( "ActivateFaces", [IsSimplicialSurface and IsNotEdgeRamified, IsRecord] );
+#! @Arguments surface, printRecord
+DeclareOperation( "DeactivateFaces", [IsSimplicialSurface and IsNotEdgeRamified, IsRecord] );
+#! @EndGroup
+
+#! @BeginGroup ActivateFace
+#! @Description
+#! Activate the face in position <A>i</A> of <A>Faces(surface)</A>. If a face is active, then the face is shown in the animation as an area. (For more information look at the start of the section <Ref Subsect="Section_LabelVisibility"/>)
+#! @Returns the updated print record.
+#! @Arguments surface, i, printRecord
+DeclareOperation( "ActivateFace", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @Arguments surface, i, printRecord
+DeclareOperation( "DeactivateFace", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @Arguments surface, i, printRecord
+DeclareOperation( "IsFaceActive", [IsSimplicialSurface and IsNotEdgeRamified, IsCyclotomic, IsRecord] );
+#! @EndGroup
+
+#! @Section Colouring
+#! @SectionLabel LabelColouring
+
+#! Finally we can configure the colours. (Colours are stored in the format 0xABCDEF where A,B,C,D,E,F are elements of the hexadecimal code.)
+#! Therefore it is possible to make the following coloured:
+#! <Enum>
+#!   <Item>Vertices (The default vertex colour is 0xF58137. 0xF58137 is an orange hue.)</Item>
+#!   <Item>Edges (The default edge colour is 0xff0000. 0xff0000 is a red hue.)</Item>
+#!   <Item>Faces (The default face colour is 0xFFFF00. 0xFFFF00 is an yellow hue.)</Item>
```

Comment:
Put the method names here as well. You can shorten the explanation in the brackets: (The default colour is .., an ... hue)